server/runtime/src/main/java/org/infinispan/server/SecurityActions.java

```diff
@@ -76,4 +77,12 @@ static void startProtocolServer(final ProtocolServer server, final ProtocolServe
    static GlobalConfiguration getCacheManagerConfiguration(EmbeddedCacheManager manager) {
       return doPrivileged(new GetCacheManagerConfigurationAction(manager));
    }
+
+   static void shutdownCache(DefaultCacheManager manager, String name) {
```

Comment:
`EmbeddedCacheManager` instead of `DefaultCacheManager`
To be consistent with the other methods.

Reply:
Done